lib/base-compiler.ts

```diff
@@ -1310,7 +1310,7 @@ export class BaseCompiler {
         } catch (e) {
             // Ignore errors
         }
-        return this.postProcess(asmResult, outputFilename, filters);
+        return await this.postProcess(asmResult, outputFilename, filters);
```

Comment:
I guess these `await` changes make sense (though is orthogonal to the rest of the changes?) Or did you find you needed it?

Reply:
Made these changes for while I was figuring out which methods I should override in `nvcc.js`. Always good form to `return await` any result promises in async methods.